src/main/java/io/appform/functionmetrics/FunctionMetricsManager.java

```diff
@@ -15,25 +16,37 @@
 
     private static MetricRegistry registry;
     private static String prefix;
+    private static boolean enableParameterCapture;
 
     private FunctionMetricsManager() {}
 
-    public static void initialize(String packageName, MetricRegistry registry) {
+    public static void initialize(String packageName, MetricRegistry registry, boolean enableParameterCapture) {
```

Comment:
Let's create an options object with defaults with a proper builder. Put `enableParameterCapture` as the sole member now. All options can be set and passed into this. Also set a default delegate with only the first two params.

Reply:
Makes sense. This should ideally be an option and clients should not be forced to set a value for this. I have added a OptionsBuilder for the same and made a method which initializes Options with default constructor and delegates to this method.